udata/tests/api/test_user_api.py

```diff
@@ -153,6 +153,8 @@ def test_suggest_users_api_no_dedup(self):
 
     def test_user_api_full_text_search_first_name(self):
```

Comment:
Could you add a test that we cant get user list without the login as admin? :pray:

Reply:
Yes, sorry!